openapi-cli/src/main/resources/templates/client/client-ep.mustache

```diff
@@ -10,46 +10,61 @@ public client class {{cut info.title " "}}Client {
     public {{cut info.title " "}}ClientConfig config;
 
     public function __init({{cut info.title " "}}ClientConfig config) {
-        http:Client httpEp = new(config.serviceUrl, {auth: config.clientConfig.auth, cache: config.clientConfig.cache});
+        http:Client httpEp = checkpanic new(config.serviceUrl, {auth: config.clientConfig.auth, cache:
+        config.clientConfig.cache});
```

Comment:
Can we indent this line by 1 tab(4 spaces) ?

Reply:
in line 13  has 1 tab space